qcengine/programs/gamess/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .runner import GAMESSExecutor
```

Comment:
It doesn't seem we need a whole folder for this? No need to increase the depth if we can add everything to a single file.

Reply:
The ones with parsing are going to get really long. What’s here for gamess is only half of a program that does  a few methods. I’d rather reconsider file layout at the end.